questions.txt

```diff
@@ -19,3 +19,78 @@ correct: $C\mathrm{e}^t$
 incorrect: $C\sin{t}$
 incorrect: $C\cos{t}$
 incorrect: $C\log{t}$
+
+------------------ Question
+
+type: TF
+prompt: The matrix, 
+<$A = \begin{bmatrix}
+<1 & 2 & 3\\
+<2 & 9 & 2\\
+<4 & 8 & 12\end{bmatrix}$, has rank 2.
+answer: true
+
+------------------ Question
+
+type: ESS
+prompt: Explain why the matrix, 
+\$A = \begin{bmatrix}
+\1 & 2 & 3\\
+\2 & 9 & 2\\
+\4 & 8 & 12\end{bmatrix}$, will have an eigenvalue equal to 0.
+example: exceptionally well!
+
+------------------ Question
+
+type: ORD
+prompt: $X$ follows a continuous distribution and has probability density
+\function
+>\[\begin{array}{l}f_X(x) = \frac{1}{2}x, \hspace{10pt} 0 \leq X \leq 2\\
+>\hspace{30pt}= 0, \hspace{10pt} \mathrm{elsewhere.}\end{array}\]
+>Order the following probability measures with the largest at the top.
+answer: The mode of $X$
+answer: The median of $X$
+answer: The mean of $X$
+
+------------------ Question
+
+type: MAT
+prompt: Match the letters to their position in the alphabet
+match_a: a
+match_b: 1
+match_a: b
+match_b: 2
+match_a: c
+match_b: 3
+
+------------------ Question
+
+type: FIL
+prompt: Upload a file please
+
+------------------ Question
+
+type: NUM
+prompt: Out of 10 how good is this question
```

Comment:
?

Reply:
If you're asking why its not maths, as I said, theres still some more questions to write but if there isn't a hurry I'll do it tomorrow.

Reply:
There's no hurry. I was pointing out the missing question mark

Reply:
gotcha solved the issue of the missing question mark